src/api/index.js

```diff
@@ -32,7 +32,15 @@ function fetchIndexerIntents() {
         if (!response.ok) {
           reject(response.statusText)
         }
-        return response.json().then(intents => intents.map(intent => ({ ...intent, makerAddress: intent.address })))
+        return response.json().then(intents =>
+          intents.map(intent => {
+            // TODO: remove address after swapVersion 2 is properly added to the indexer
+            if (intent.address === '0xbabe31056c0fe1b704d811b2405f6e9f5ae5e59d') {
+              return { ...intent, makerAddress: intent.address, swapVersion: 2 }
+            }
+            return { ...intent, makerAddress: intent.address }
```

Comment:
can you explain what's going on here?

Reply:
@perich Yeah my TODO comment could have been more descriptive, swapVersion property hasn’t been added to the indexer yet so I’m hardcoded my maker address as a swapV2 maker until the ability intents can be set with that property.